src/uct/ib/ud/accel/ud_mlx5.c

```diff
@@ -721,7 +721,7 @@ static UCS_CLASS_INIT_FUNC(uct_ud_mlx5_iface_t,
 
     ucs_trace_func("");
 
-    init_attr.res_domain_key = UCT_IB_IFACE_NULL_RES_DOMAIN_KEY;
+    init_attr.res_domain_key = UCT_IB_MLX5_RES_DOMAIN_KEY;
```

Comment:
this cause to use different keys for ud_mlx5/ib/cm... is it ok?

Reply:
verbs should have NULL key since it does not use res domain
and all mlx5 transports should have same non-NULL key